headers/functions/arm9.h

```diff
@@ -317,6 +317,9 @@ int LoadWanTableEntryFromPackUseProvidedMemory(struct wan_table* wan_table,
 int ReplaceWanFromBinFile(struct wan_table* wan_table, int wan_id, enum pack_file_id pack_id,
                           uint16_t file_index, bool compressed);
 void DeleteWanTableEntryVeneer(struct wan_table* wan_table, int wan_id);
+bool DoesWanHaveAnimationGroup(struct wan_header* wan_header, uint16_t animation_group_id);
+void DoesSpriteInWanTableHaveAnimationGroup(uint16_t sprite_id, uint16_t animation_group_id);
```

Comment:
Can we shorten this to `WanTableSpriteHasAnimationGroup`? Also, the return type should be `bool`?